Recommend authentication via ssh key with ed25519 algorithm (#231) (#291)
deltachat/chatmail: README.md

## Deploying your own chatmail server 

> To deploy chatmail on your own server, you must have set-up ssh authentication and need to use an ed25519 key, due to an [upstream bug in paramiko](https://github.com/paramiko/paramiko/issues/2191). You also need to add your private key to the local ssh-agent to avoid having to type the password for your key multiple times.

Reviewer: ```suggestion
To deploy chatmail on your own server, you must have set-up ssh authentication and need to use an ed25519 key, due to an [upstream bug in paramiko](https://github.com/paramiko/paramiko/issues/2191). You also need to add your private key to the local ssh-agent, because you can't type in your password during deployment.
```

Otherwise some might think "oh, I'm fine with entering my password multiple times, whatever".

Author: makes sense, didn't know it wasn't possible at all, did not want to test it :D
